tools/validate_configs/test_configs/hpc-cluster-small-slurm-v5.yaml

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: hpc-cluster-slurm-v5
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: hpc-small-v5
+  region: us-central1
+  zone: us-central1-c
+  slurm_cluster_name: ghpc
+
+deployment_groups:
+- group: primary
+  modules:
+  # Source is an embedded resource, denoted by "resources/*" without ./, ../, /
+  # as a prefix. To refer to a local resource, prefix with ./, ../ or /
+  # Example - ./resources/network/vpc
+  - source: modules/network/vpc
+    kind: terraform
+    id: network1
+
+  - source: modules/file-system/filestore
+    kind: terraform
+    id: homefs
+    use: [network1]
+    settings:
+      local_mount: /home
+
+  - source: community/modules/compute/schedmd-slurm-gcp-v5-partition
+    kind: terraform
+    id: debug_partition
+    use:
+    - network1
+    - homefs
+    settings:
+      partition_name: debug
+      node_count_dynamic_max: 4
+      enable_placement: false
+      machine_type: n2-standard-2
+      is_default: true
```

Comment:
What happens if this is not set on any partition? Does it resolve gracefully? This is a new requirement for v5 and I could imagine customers leaving it off.

Reply:
There would not be a default partition, which is entirely valid, but could be a bit annoying.

My preference would be to set the first listed partition as default, but ran into some errors when I tried to access and update an element of the list in terraform from the controller module, something similar to:
```
The "for_each" value depends on resource attributes that cannot be determined until apply, so Terraform cannot predict how many instances will be created.
```

I'll spend a bit more time on this and see if I can find another workaround.